### DISCUSSION TITLE
Show in the chatbox when you're silenced

### DISCUSSION THREAD

**Discussion by Atarizu** [ppy/osu#24526] [Ideas]
This is a lazer thing, but I took a mute and noticed lazer still had the "type a message" even though stable shows I'm silenced. I suppose there was no one brave out there to take a mute and report this?

lazer:
![lazer](https://github.com/ppy/osu/assets/133982507/708f4969-021a-450a-b84e-ead09541a5fa)

stable:
![stable](https://github.com/ppy/osu/assets/133982507/8b6de590-adf8-4d86-bbc2-be3e5b149fea)

**Comment by Userlog2** [comment]
I have an idea for a design:

<img width="1920" height="1080" alt="osu-lazer-silence-display-idea" src="https://github.com/user-attachments/assets/97854272-1663-456e-83a0-f81798854a44" />

- The whole chat bar would turn the same yellow as the "Silence" thing on web
- **The bright yellow colour would drain away and be replaced by a darker yellow as the silence progresses**
  - For example, if 5 minutes of a 10-minute silence has passed, half the bar would be bright yellow and the other half would be dark yellow
- There would be a message like "You have been [silenced](https://osu.ppy.sh/wiki/en/Help_centre/Silences) for `reason` and will be able to talk again in `time`."
  - The "silenced" text would be a link to the wiki article
  - The `reason` would just be the description visible on web
  - The `time` would display in days, hours, minutes, and seconds where relevant

Note that my design is an extremely rough draft meant to show how the UX might be.
